### PR TITLE
add new OpenDUT

### PR DESCRIPTION
**Pull request by madri2**

### CODE REVIEW

File: BB-EST/Testing/BB_OpenDUT.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# OpenDUT 
+
+## BB Tags(s)
+
+- EST, CEST, SC-TC, CSC-TC
```
Comment (LS0205) [inline BB-EST/Testing/BB_OpenDUT.md:5]: Bitte Tags auf - BB-EST, BB-CEST, BB-SC-TC, BB-CSC-TC ändern

File: BB-EST/Testing/BB_OpenDUT.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Functional Clusters
+
+- (Explanation: in which Functional Cluster the BB be located; if none of the existing fit new required)
+
+
+## Layer
+
+- Middleware
```
Comment (LS0205) [inline BB-EST/Testing/BB_OpenDUT.md:14]: Auf "MWLayer" ändern

**Review by LS0205 — CHANGES_REQUESTED** [review]
please rework it as suggested by the comments

File: BB-EST/Testing/BB_OpenDUT.md (hunk 1 above)
Comment (madri2) [reply BB-EST/Testing/BB_OpenDUT.md]: done

File: BB-EST/Testing/BB_OpenDUT.md (hunk 2 above)
Comment (madri2) [reply BB-EST/Testing/BB_OpenDUT.md]: Done

**Review by LS0205 — APPROVED** [review]
ok

**Pull request merged**